### ISSUE TITLE
[Masthead/Footer] Accessibility - The 'role' defined on the element is not valid per WAI-ARIA specification - DDS Consulting

### ISSUE THREAD

**Issue by abhin07** [carbon-design-system/carbon-for-ibm-dotcom#6890]
IBM Equal Access Accessibility Checker
The 'role' defined on the element is not valid per WAI-ARIA specification

Elements showing the error 
```
<dds-footer size="" data-autoid="dds--footer" aria-label="footer" role="footer">
```
----
- Hybrid Cloud developer: 
- [HC Jira ticket]
- Dev estimate:  
- DDS consultant:

**Comment by RobertaJHahn** [comment]
@Praveen-sr Could you work with Ram to see whether this bug identified by the IBM Equal Access Accessibility Checker is valid? If it is, could you let us know what the priority would be. Thanks.

**Comment by Praveen-sr** [comment]
Expected fix: Here the role "footer" is not valid. We have raised a defect on absence of footer region in screen reader testing. The footer section should be given within either html5 footer tag or aria contentinfo role. 
Note: Assign these roles on a container like a div not on any other elements.

**Comment by proeung** [comment]
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2149
HC Engineer Assignee: @andy-blum
